bioio_ome_tiff/reader.py

```diff
@@ -424,6 +472,43 @@ def physical_pixel_sizes(self) -> types.PhysicalPixelSizes:
         """
         return physical_pixel_sizes(self.metadata, self.current_scene_index)
 
+    @property
+    def micro_manager_metadata(self) -> Dict[str | int, Any]:
+        """
+        Returns
+        -------
+        micro_manager_metadata: dict[str|int, Any]
+            Expose the unprocessed metadata from the file, additionally
+            flattening Adobe private tag 50839 into the top level of the dict.
+        Notes
+        -----
+        this is in response to a user request:
+            https://github.com/bioio-devs/bioio-ome-tiff/issues/5
+        """
+        if self._micro_manager_metadata is not None:
+            return self._micro_manager_metadata
+
+        self._micro_manager_metadata = {}
+        with self._fs.open(self._path) as open_resource:
+            with TiffFile(open_resource, is_mmstack=False) as tiff:
+                # Get unprocessed metadata from tags
+                tiff_tags = self._get_tiff_tags(tiff)
+                micro_manager_metadata = {}
+                for k, v in tiff_tags.items():
+                    # break up code 50839 which is where it seems MM metadata lives
+                    # 50839 is a private tag registered with Adobe
+                    if k == 50839:
+                        try:
+                            for kk, vv in json.loads(v["Info"]).items():
+                                micro_manager_metadata[kk] = vv
+                        except Exception:
+                            # if we can't parse the json, just ignore it
+                            pass
+                    else:
+                        micro_manager_metadata[k] = v
```

Comment:
I don't know that it makes sense to call this the micromanager metadata.  Why not only return the contents of tag 50839, or None if it isn't present?    Unless I am missing something...

Reply:
https://github.com/AllenCellModeling/aicsimageio/issues/349 asked for more than just the content of 50839.  I thought there was a meeting where that name was decided; it seems like maybe the note I made recording the conclusion of that meeting ( https://github.com/bioio-devs/bioio-ome-tiff/issues/5#issuecomment-2752666008 ) might have been mistaken.  Do you have a suggestion for a name that makes more sense?

Reply:
I think the name makes sense if the scope of metadata output by this was narrowed down like Dan is suggesting, right now it seems to include more than micromanager metadata

Reply:
how about 'unprocessed_metadata'?

Reply:
I think that might be confusing for future development since the `.metadata` fields pulls the `UNPROCESSED_METADATA` block from the `xarray` - I think narrowing the scope of this method so that it only does micromanager metadata like the ticket needed access to would be ideal.

Reply:
We may need a meeting to discuss this, I thought the phrase "# **also** break up code 50839 which is where it seems MM metadata lives". implies the request was for MORE than just the contents of 50839.

Reply:
Yeah that is true, but this does process data in some form which is against the definition of unprocessed metadata we have elsewhere so I think the simplest way to avoid this complication is by providing easy access to the micromanager metadata through this access point and allow the unprocessed metadata to remain unprocessed.

Reply:
@kmitcham We already have methods that return all the tiff tags.  
My comment here was meant to say that you should not return all the tiff tags as micromanager metadata.  I wanted you to restrict it to tag 50839 only, as opposed to renaming anything.  I think we already return all the tiff tags in the unprocessed (or is it the processed?) metadata, so if people want a raw tiff tag, they can already get it that way.  I thought the main ask was just to unpack 50839 (which is the micromanager-specific one) into some kind of dict that we can hand back to the user.

Reply:
I understood the ticket to be asking for all the metadata including the MM data.  I am updating the code to Dan's interpretation.